frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -36,6 +36,224 @@ const connector = connect(
   }
 );
 
+// TODO: this button is placed at the bottom of the menu,
+// But I am able to scroll down.. so it's not at the bottom.
+// If someone could help with the height stuff so there's no scrolling that would be great
+const SignOutButton = styled.button`
+  position: absolute;
+  bottom: 0px;
+
+  background-color: #fff0;
+  border-width: 0px;
+  color: ${INK};
+  font-weight: bold;
+
+  :hover {
+    cursor: pointer;
+  }
+
+  :active {
+    opacity: 0.8;
+  }
+`;
+
+const DashboardContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  min-height: 100%;
+  min-width: 100%;
+  padding-top: 98px;
+
+  @media (max-width: 600px) {
+    flex-direction: column;
+    padding-right: 0px;
+  }
+`;
+
+const MenuContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100%;
+  min-width: 309px;
+  border-radius: 0px 12px 0px 0px;
+
+  padding: 64px 64px 64px 64px;
+
+  background: ${CLOUD};
+
+  @media (max-width: 600px) {
+    width: 100%;
+    min-height: 347px;
+    border-radius: 0px 0px 0px 0px;
+    padding: 48px 24px 24px 24px;
+  }
+`;
+
+// HELP
+// Thinking of making a 2-column grid with the left-hand column being fixed-width for icons
```

Comment:
I think this is good for now, and we can have someone work on the menu item specifically!